test/components/index.js

```diff
@@ -0,0 +1,52 @@
+/*!
+ * Copyright (c) 2019 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+import * as brVue from 'bedrock-vue';
+import Vue from 'vue';
+import VueRouter from 'vue-router';
+
+// FIXME: make configurable
+Vue.config.devtools = true;
+
+// install all plugins
+Vue.use(brVue);
+
+brVue.setRootVue(async () => {
+  // load dynamic imports in parallel
+  const [
+    brQuasar,
+    Quasar,
+    {default: iconSet},
+    {default: Vuelidate}
+  ] = await Promise.all([
+    import('bedrock-quasar'),
+    import('quasar-framework'),
+    import('quasar-framework/icons/fontawesome'),
+    import('vuelidate')
+  ]);
+
+  // replace default `br-root` with a custom one
+  Vue.component('br-root', () => import('./BrRoot.vue'));
+
+  // install all Vue plugins
+  Vue.use(Vuelidate);
+
+  const router = new VueRouter({
+    mode: 'history',
+    routes: [{
+      path: '/',
+      component: () => import('./Home.vue'),
+      meta: {title: 'Bedrock Wizard'}
+    }]
+  });
+
+  // configure quasar
+  Quasar.icons.set(iconSet);
+
+  const BrApp = Vue.component('br-app');
+  return new BrApp({
```

Comment:
One line?

Reply:
@mattcollier Usually, I'd say one line. The multiline pattern is a common pattern when creating a new Vue instance. In this special case, multiline is fine with me.